test/assignment5/sockettest.sh

```diff
@@ -81,7 +83,19 @@ if [ -e long_string.txt ]; then
     sendstring=`cat long_string.txt`
     test_send_socket_string ${sendstring} ${comparefile}
 fi
+
+if [ "$multi_line_string_test" = true ]; then
+    echo "Sending Multiple packets from multiple_packets.txt file"
```

Comment:
@DeekshithPatil I think this should no longer reference `multiple_packets.txt` file?